src/runtime/interrupt/interrupt_k210.go

```diff
@@ -0,0 +1,19 @@
+// +build k210
```

Comment:
This file may need to be extended since https://github.com/tinygo-org/tinygo/pull/1171 got merged.

Reply:
Looking at #1171 and the `interrupt_sifive.go` file I think this is correct but I may be missing something.

Reply:
Is this part now working as expected?

Reply:
Yes, interruptions seem to work.